src/campaign.ts

```diff
@@ -0,0 +1,7 @@
+export interface Campaign {
```

Comment:
store your interfaces in src/interfaces/yourfile

Reply:
done